package.json

```diff
@@ -39,6 +39,7 @@
     "@docusaurus/preset-classic": "^3.1.0",
     "@docusaurus/remark-plugin-npm2yarn": "^3.1.0",
     "@docusaurus/theme-mermaid": "^3.1.0",
+    "@kiltprotocol/sdk-js": "1.0",
```

Comment:
```suggestion
```

Not needed here

Reply:
Fixed in: https://github.com/KILTprotocol/docs/pull/329/commits/2fb4bf229be1bd8c28ed45066790a66d8b5f1e29